#!/usr/bin/python3

def raise_exception_msg(message=""):
    """Raise exception with messge."""
    raise NameError(message)
